integration/integration_test.go

```diff
@@ -896,10 +955,10 @@ func initIntegrationTestConfig() *integrationTestConfig {
 	var gcsEndpoint string
 	var disableGcsAuth bool
 	flag.StringVar(&c.gcsBucket, "bucket", "gs://kaniko-test-bucket", "The gcs bucket argument to uploaded the tar-ed contents of the `integration` dir to.")
-	flag.StringVar(&c.imageRepo, "repo", "gcr.io/kaniko-test", "The (docker) image repo to build and push images to during the test. `gcloud` must be authenticated with this repo or serviceAccount must be set.")
+	flag.StringVar(&c.imageRepo, "repo", "localhost:5000", "The (docker) image repo to build and push images to during the test. `gcloud` must be authenticated with this repo or serviceAccount must be set.")
```

Comment:
I missed this initially but can  you also change this back to its original value